### DISCUSSION TITLE
Tapping on a shortcut should allow removing/sharing it

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53969] [Mobile] [answered]
GitHub Mobile beta
![Screenshot_20230428-093320.png](https://user-images.githubusercontent.com/2119212/235161941-aa2d62f5-7040-4725-9311-b79509e77933.png)
- (For information about this specific shortcut, see https://github.com/community/community/discussions/53966.)

Apparently in some places GitHub Mobile supports tap and hold to do something. This isn't one of those cases, but it's definitely a place I'd expect it to be available. 

Specifically, tap and hold should let me remove a shortcut and it should let me copy/share the link.

**Comment by taki-on** [comment, marked as answer]
Thank you for your suggestion :) We will discuss this internally.
